overlord/managers_test.go

```diff
@@ -1843,6 +1844,7 @@ type: kernel`
 
 	c.Assert(bloader.BootVars, DeepEquals, map[string]string{
```

Comment:
we should probably make BootVars unexported at some point indeed, but not an immediate concern, so many other things

Reply:
yes agreed